experimental/devices/ina219/ina219doc.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2016 The Periph Authors. All rights reserved.
```

Comment:
Update year. I'd recommend `doc.go` since this is what is generally used in other packages.